src/foam/nanos/auth/Address.js

```diff
@@ -51,31 +51,20 @@ foam.CLASS({
     {
       class: 'String',
       name: 'address1',
-      // required: true
       width: 70,
       displayWidth: 50,
       documentation: 'An unstructured field for the main postal address.',
-      validateObj: function(address1) {
-        var address1Regex = /^[a-zA-Z0-9 ]{1,70}$/;
-
-        if ( address1.length > 0 && ! address1Regex.test(address1) ) {
-          return 'Invalid address line.';
-        }
-      }
+      expression: function(structured, streetNumber, streetName) {
```

Comment:
Are we truly using both structured and unstructured addresses in the system?

Reply:
Don't know off the top of my head and I'd love to remove this stuff but I'm afraid of non-backwards compatible changes right now.

Reply:
Yeah me too, I hear you.  Backwards compatability is what has me nervous about adding this expression, but I think this is probably the best idea at the moment.